debian/pop-plymouth-theme.debhelper.log

```diff
@@ -0,0 +1,15 @@
+dh_update_autotools_config
```

Comment:
`debian/pop-plymouth-theme` is generated during build and shouldn't be committed.

Reply:
ya that was a goof. I was going to get around to that once I get the packaging to even work.

Reply:
I added them to a `.gitignore` to remove those files. Anything that I missed?